README.md

```diff
@@ -224,6 +203,31 @@ dotnet script -d foo.csx -- -d
 ```
 
 will pass the `-d` before `--` to `dotnet script` and enable the debug mode whereas the `-d` after `--` is passed to script for its own interpretation of the argument.
+
+## Extras
+
+Beyond the standard scripting dialect support (compatible with `csi.exe`), `Dotnet.Script` provides some extra features, located in the `Dotnet.Script.Extras` package.
```

Comment:
at the moment they are not. 
Once #55 is resolved there will be a discovery process in place.
Note that this is not released at the moment anywhere, so nothing is broken.